echopype/convert/api.py

```diff
@@ -282,7 +282,7 @@ def _save_groups_to_file(echodata, output_path, engine, compress=True):
             path=output_path,
             mode='a',
             engine=engine,
-            group='Beam',
+            group='Beam_complex',
```

Comment:
for line 279: why do you have to check `hasattr(echodata, "beam_complex")`? From [here](https://github.com/OSOceanAcoustics/echopype/blob/aa9c32f205ae7aeaded217800119b59ef6a175ec/echopype/echodata/echodata.py#L98-L101) it looks like _all_ groups are set to None when EchoData is initialized, regardless of the type of sonar_model. Did I miss anything?